elyra/pipeline/parser.py

```diff
@@ -128,7 +131,7 @@ def _get_pipeline_definition(pipeline_definitions: Dict, pipeline_id: str) -> [D
         return None
 
     @staticmethod
-    def _create_pipeline_operation(node: Dict, super_node: Optional[Dict] = None):
+    def _create_pipeline_operation(root_dir: str, node: Dict, super_node: Optional[Dict] = None):
```

Comment:
Per previous comment, both this method and `_get_remaining_component_params()` would not be static and `_get_remaining_component_params()` can reference `self.root_dir` directly.